Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -447,3 +466,52 @@ def create(
             )
             return None
         return self.issue(response.json()['id'])
+
+    def cc_radar(self, issue, block=False, timeout=None):
+        if not self.radar_importer:
+            sys.stderr.write('No radar importer specified\n')
+            return None
+
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, RadarTracker):
+                break
+        else:
+            sys.stderr.write('Project does not define radar tracker\n')
+            return None
+
+        did_modify_cc = False
+        if self.radar_importer not in issue.watchers:
+            log.info('CCing {}'.format(self.radar_importer.name))
+            response = None
+            try:
+                response = requests.put(
+                    '{}/rest/bug/{}{}'.format(self.url, issue.id, self._login_arguments(required=True)),
+                    json=dict(
+                        ids=[issue.id],
+                        cc=dict(add=[self.radar_importer.username]),
+                    ),
+                )
+            except RuntimeError as e:
+                sys.stderr.write('{}\n'.format(e))
+            if response and response.status_code // 100 == 4 and self._logins_left:
+                self._logins_left -= 1
+            if response and response.status_code // 100 == 2:
+                did_modify_cc = True
+                issue._comments = None
+                issue._references = None
+            else:
+                sys.stderr.write("Failed to cc {} on '{}'\n".format(self.radar_importer.name, issue))
+
+        start = time.time()
+        while start + (timeout or 60) > time.time():
+            for reference in issue.references:
+                if isinstance(reference.tracker, RadarTracker):
+                    return reference
+            if not block or not did_modify_cc:
+                break
+            log.warning('Waiting until {} imports bug...'.format(self.radar_importer.name))
+            time.sleep(10)
+            issue._comments = None
+            issue._references = None
+
```

Comment:
Should we write some error message when we exhaust timeout?

Reply:
Good idea, added that.